src/main/g8/default.properties

```diff
@@ -1,6 +1,6 @@
 name = My Kalix Project
 description = Creates a Kalix project with a Value Entity
-sdk_version = maven(io.kalix, kalix-java-sdk)
+sdk_version = maven(io.kalix, kalix-jvm-core-sdk)
```

Comment:
This trick to pick the latest sdk version was broken after we released 1.2.0. 

We don't publish `kalix-java-sdk` anymore. Moving to `kalix-jvm-core-sdk` as it should be more "stable".

Reply:
nice 👍